assets/js/mentors.js

```diff
@@ -136,6 +136,20 @@ var controllerMentors = (function(jQuery) {
         });
     };
 
+    jQuery("#copy-link").click(function () {
```

Comment:
check the code on this page, you should follow the current style of implementing. Both ways works, but would be better to keep like that: 

``` 
var copyLink = jQuery('#link-msg');

copyLinkt.click(function() {
     ... 
} 
```

Reply:
Thank you for pointing that out. I'll make sure to follow the current style in the future. Apologies for the oversight! 🙃